packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -589,6 +590,12 @@ export type CallParticipantsLocator = {
     participantIDs: string[];
 };
 
+// @beta
+export type CallPermissionOptions = {
```

Comment:
Two naming comments:

* Similar to above, `CallPermission` is too generic. There are multiple types of permissions in the context of a call.
  * Also, avoid the suffix `Options`. Many optional structs are... options. And this one sits inside a `CallCompositeOptions` already.
* It might also be confusing that there are multiple layers that require / check for device permissions. Headless calling already has two related types: `PermissionsConstraint` and `DeviceAccess`.

How about

```typescript
interface DevicePermissionPrompts {
  camera: 'promptAndBlock' | 'prompt' | 'doNotPrompt';
  microphone: 'promptAndBlock' | 'prompt' | 'doNotPrompt';
};
```